playwright/utils/graphqlHelpers.ts

```diff
@@ -0,0 +1,96 @@
+import { request } from '@playwright/test';
+
+const API_KEY = 'da2-fakeApiId123456';
+const GRAPHQL_URI =
+  process.env.AWS_APPSYNC_GRAPHQL_URL || 'http://localhost:20002/graphql';
```

Comment:
let's revisit these keys, can we move it to `.env` or this one is expected?

Reply:
> let's revisit these keys, can we move it to .env or this one is expected?

it's copied from the scripts we already have  in, e.g. `scripts/create-colony.ts`